### PR TITLE
Change XR Android export to mention OpenXR vendors plugin

### PR DESCRIPTION
**Pull request by BastiaanOlij**
We've renamed the OpenXR loaders plugin to vendors plugin as it will be containing more than just the loaders. This PR changes the manual to reflect this.

### CODE REVIEW

File: tutorials/xr/deploying_to_android.rst [hunk 1]
```diff
@@ -33,34 +33,35 @@ This will create a folder called **android** inside of your project that contain
...
 
 .. warning::
-    The Android plugin structure has been restructured in Godot 4.2, if you've previously installed the loader plugin you need to delete it from the `android/plugins` folder.
-    The loader plugin available in the asset library is still the old one, it will be updated to the new one after 4.2 is released.
+    The Android plugin structure has been restructured in Godot 4.2, and the loader plugin was renamed to vendors plugin as it now includes more than just loaders.
+    If you've previously installed the loader plugin you need to delete it from the `android/plugins` folder.
+    The loader plugin available in the asset library is still the old one, it will be updated to the new vendor plugin after 4.2 is released.
 
-The loaders can be downloaded from the asset library, search for OpenXR Loaders and install the plugin:
+The vendors plugin can be downloaded from the asset library, search for OpenXR and install the plugin:
```
Comment (BastiaanOlij) [inline tutorials/xr/deploying_to_android.rst:44]: Note, I'm still showing the old screenshot here as the new vendors plugin isn't in the asset library yet. We will be updating this part of the manual, and removing the warning up above, once the plugin is available in the asset library (likely after the Godot 4.2 official release)

File: tutorials/xr/deploying_to_android.rst
Comment (m4gr3d) [inline tutorials/xr/deploying_to_android.rst:1]: Looks good!

Can you also update the OpenJDK version to 17; this reflects the update in the Android build documentation.

```
- Installing OpenJDK 17
```

**Review by m4gr3d — APPROVED** [review]

**Review by skyace65 — APPROVED** [review]

File: tutorials/xr/deploying_to_android.rst
Comment (BastiaanOlij) [reply tutorials/xr/deploying_to_android.rst:1]: Done

**Comment by skyace65** [comment]
Thanks!

**Pull request merged**